app/build.gradle

```diff
@@ -243,6 +243,7 @@ dependencies {
     compile "io.reactivex.rxjava2:rxjava:${RX_JAVA_VERSION}"
     //compile 'net.ltgt.dagger:bullet:1.0-SNAPSHOT' // Loaded from JitPack
     compile "org.parceler:parceler-api:${PARCELER_VERSION}"
+    compile files('libs/YouTubeAndroidPlayerApi.jar')
```

Comment:
You were right Santi! works also with only this line in the gradle `compile fileTree(include: ['*.jar'], dir: 'libs')`.